pkg/controller/clone-controller.go

```diff
@@ -733,6 +723,7 @@ func MakeCloneSourcePodSpec(sourceVolumeMode corev1.PersistentVolumeMode, image,
 			{
 				Name:      DataVolName,
 				MountPath: common.ClonerMountPath,
+				ReadOnly:  true,
```

Comment:
Nice, is there a real impact though? IIUC the other ReadOnly is just about enforcing this one